src/sss_client/pam_sss.c

```diff
@@ -2306,14 +2430,48 @@ static int pam_sss(enum sss_cli_command task, pam_handle_t *pamh,
                 }
                 break;
             case SSS_PAM_CHAUTHTOK:
+                /*
```

Comment:
I don't understand why was this code added? Is it to prevent pam_sss from prompting in the case there was already a password on the stack (use_first_pass) or as another case to make sure that pam_sss prompts even with a password if PROMPT_ALWAYS is used and therefore to make sure rest of the added code is called?